reproject/mosaicking/coadd.py

```diff
@@ -22,6 +22,9 @@ def reproject_and_coadd(
     combine_function="mean",
     match_background=False,
     background_reference=None,
+    final_array=None,
+    final_footprint=None,
```

Comment:
Maybe it would make sense to simply use ``output_array`` and ``output_footprint`` for consistency with the individual reproject functions.

Reply:
I like that idea; `final_array` was just an internal use anyway.